src/boot/efi/pe.h

```diff
@@ -20,13 +20,13 @@ typedef struct PeSectionHeader {
 /* This is a subset of the full PE section header structure, with validated values, and without
  * the noise. */
 typedef struct PeSectionVector {
-        size_t size;
+        size_t memory_size;
```

Comment:
please add a brief comment here, explaining the situation (i.e. a brief version of what you wrote in the commit msg)

just to match the siilar comments for the two other fields below